src/Mod/Draft/draftfunctions/upgrade.py

```diff
@@ -507,7 +507,7 @@ def makeWires(objectslist):
             elif len(objects) > 1 and len(edges) > 1:
                 result = makeWires(objects)
                 if result:
-                    if result == True:
+                    if result:
```

Comment:
```suggestion
                if makeWires(objects):
                    _msg(translate("draft","Found several wires or edges: wiring them"))
                else:
                    _msg(translate("draft","Found several non-treatable objects: creating 
```
Bonus points for refactoring this

Reply:
done